extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -157,14 +159,69 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
       : FALLBACK_DEFAULT_STICKER;
   }
 
+  /**
+   * @private
+   */
+  maybeInitializeStickerStyle_() {
+    this.toggleStickerState_(
+      'outline',
+      this.element.getAttribute('sticker-style') === 'outline'
```

Comment:
Get the attribute at the top of the function so you don't have to get it multiple times.

Reply:
Obsolete.